src/pages/developing/angular-tutorial/step-1.mdx

```diff
@@ -301,34 +301,46 @@ package.
 </InlineNotification>
 
 #### Import and register icons
-
-Now let's import the icons from our `@carbon/icons` package. In `app.module.ts`,
-`app.component.spec.ts` and `header.component.spec.ts`, we need to import each
-individual icon we will use and register them with the `IconService`
+Now let's import the icons from our `@carbon/icons` package. To not make our app very big, you don't need to import
+all the icons just those you would like to use. To do so let's open the `header.component.ts` file and import `Notification20` icon like below.
 
 <!-- prettier-ignore-start -->
-```javascript path=src/app/app.module.ts,src/app/app.component.spec.ts,src/app/header/header.component.spec.ts
+```javascript path=src/app/header/header.component.ts
 import Notification20 from '@carbon/icons/es/notification/20';
-import UserAvatar20 from '@carbon/icons/es/user--avatar/20';
-import AppSwitcher20 from '@carbon/icons/es/app-switcher/20';
 ```
 <!-- prettier-ignore-end -->
 
-In the `AppModule` class we'll add a constructor that provides us with an
-instance of `IconService` and call `registerAll` with an array of the icons we
-need to use.
+Now you need to register the icon via `IconService` that also needs to be imported from `carbon-components-angular` module.
+After importing IconService you need to inject it in component constructor and us it in OnInit life cycle component hook. There are 2 methods for icon registration `.register()` which accepts only one icon and `.registerAll()` which accepts array of icons. As we are going to use more than one icon we are going to use the later method as below.
+
+<!-- prettier-ignore-start -->
+```javascript path=src/app/header/header.component.ts
+import { IconService } from "carbon-components-angular";
+...
+
+constructor(protected iconService: IconService) {}
 
-```javascript path=src/app/app.module.ts,src/app/app.component.spec.ts,src/app/header/header.component.spec.ts
-constructor(protected iconService: IconService) {
-  iconService.registerAll([
-    Notification20,
-    UserAvatar20,
-    AppSwitcher20
-  ]);
-}
+  ngOnInit() {
+    this.iconService.registerAll([Notification20]);
+  }
 ```
+<!-- prettier-ignore-end -->
+
+Next step is to import the `IconModule` in the `AppModule` module where the `HeaderComponent` is declared.
+
+<!-- prettier-ignore-start -->
+```javascript path=src/app/header/header.component.ts
+import { IconModule } from "carbon-components-angular";
+...
+
+imports: [
+  ...
+  IconModule
+]
+```
+<!-- prettier-ignore-end -->
 
-Then we need to add the template code. Populate `header.component.html` with:
+Now the icon is ready to be used in template code. Template in `header.component.html` should look like this:
```

Comment:
```suggestion
In your app.module.ts, import `IconService` & `IconModule` from `carbon-components-angular` module. IconModule will enable you to use the `ibmIcon` directive. `IconService` will enable you to register icons from `@carbon/icons` to be used with the directive.

<!-- prettier-ignore-start -->
```javascript path=src/app/app.module.ts
import { IconModule. IconService } from "carbon-components-angular";
<!-- prettier-ignore-end -->

Next, inject the service in the module class constructor and register it. There are 2 methods for icon registration `.register()` which accepts only one icon and `.registerAll()` which accepts array of icons. As we are going to use more than one icon, we are going to use the later method.

<!-- prettier-ignore-start -->
```javascript path=src/app/app.module.ts
export class AppModule {
  constructor(protected iconService: IconService) {
    iconService.registerAll([
      Notification20,
      UserAvatar20,
      AppSwitcher20
    ]);
  }
}
<!-- prettier-ignore-end -->
```

Breaking up information here to make it easier for users to absorb the steps & follow. Reasoning for not importing icons in header.component is given above.

Reply:
thx for suggestions. I accepted them but not able to commit last suggestion. it's too long.
I didn't clone the repo on my local and created the PR just from web editor.

Reply:
but obviously I will need to clone it as not able to run auto format. Will look into it today or following days. Thx for now.

Reply:
Great, thanks! FYI, you'll need node v16.